test/sql/join/iejoin/test_iejoin_overlaps.test

```diff
@@ -0,0 +1,21 @@
+# name: test/sql/join/iejoin/test_iejoin_overlaps.test
+# description: IEJoin block short circuiting
+# group: [iejoin]
+
+statement ok
+PRAGMA enable_verification
+
+# We read from CSVs to prevent the optimiser from
```

Comment:
Can we get some more tests for the IEJoin?

* IEJoin in subqueries
* IEJoin in recursive ctes
* Larger IE Joins and IE joins with arrays/strings as indicated above